skyportal/handlers/api/photometry.py

```diff
@@ -173,6 +173,14 @@ def standardize_photometry_data(self):
                 )
             else:
                 kind = 'flux'
+                if (
```

Comment:
The documentation of `PhotFluxFlexible` says

>     """This is one of two classes used for rendering the
    input data to `PhotometryHandler.post` in redoc. These classes are only
    used for generating documentation and not for validation, serialization,
    or deserialization."""

I guess that's not true!

Reply:
Should these checks be specified in `schema.py` using the [validate keyword](https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.Method)?

Reply:
Agreed -- updated